t/sanity.t

```diff
@@ -815,7 +815,7 @@ init_worker_by_lua '
         shm = "healthcheck",
         upstream = "foo.com",
         type = "http",
-        http_req = "GET /status HTTP/1.0\\\\r\\\\nHost: localhost\\\\r\\\\n\\\\r\\\\n",
+        http_req = "GET /status HTTP/1.0\r\nHost: localhost\r\n\r\n",
```

Comment:
@szelcsanyi It depends on what directives you are using around these. `_by_lua '...'` requires double escaping while `_by_lua_block {...}` does not require any special treatment at all. See below for more details:

https://github.com/openresty/lua-nginx-module#special-escaping-sequences